src/test/java/seedu/dengue/storage/TemporaryMemoryTest.java

```diff
@@ -0,0 +1,111 @@
+package seedu.dengue.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
```

Comment:
I would suggest importing MAX_SIZE from seedu.dengue.storage.temporary.MemoryStack to ensure that test case will not be broken if MAX_SIZE is changed